add_tasks.js

```diff
@@ -0,0 +1,198 @@
+var tasks = [];
+
+function addTask(e) {
+  if (e.keyCode === 13) {
+    tasks.unshift({
+      taskName: document.getElementsByClassName("task_add_new")[0].value,
+      isCompleted: false
+    });
+    showTaskInserted();
+    document.getElementsByClassName("task_add_new")[0].value = "";
+  }
+}
+
+function showTaskInserted() {
+  var containerAllTasks = document.body.getElementsByClassName("tasks_all")[0];
+  var newElementTask = document.createElement("div");
+  newElementTask.className = "task_active";
+  newElementTask.innerHTML =
+    '<input type="checkbox" class="task_checkbox" onchange="selectCompletedTask()"' +
```

Comment:
you have similar block in different function , maybe better to have one function which is called in different part of code?